src/mono/wasm/debugger/BrowserDebugHost/Startup.cs

```diff
@@ -112,6 +85,24 @@ string GetEndpoint(HttpContext context)
                     return $"{devToolsHost.Scheme}://{devToolsHost.Authority}{request.Path}{request.QueryString}";
                 }
 
+                async Task ReturnPageLinksToDebug(HttpContext context)
+                {
+                    HttpRequest request = context.Request;
+
+                    {
+                        Dictionary<string, string>[] tabs = await ProxyGetJsonAsync<Dictionary<string, string>[]>(GetEndpoint(context) + "json/list");
+                        context.Response.ContentType = "text/html";
+                        string urlsToInspect = "<title>Inspectable pages</title><h3>Inspectable pages</h3><hr>";
+                        foreach (var tab in tabs)
+                        {
+                            string aHref = tab["devtoolsFrontendUrl"];
+                            aHref = aHref.Replace($"ws={devToolsHost.Authority}", $"ws={request.Host}");
+                            urlsToInspect += $"<a href=\"http://{devToolsHost.Authority}{aHref}\">{tab["title"]}<br><span>{tab["url"]}</a><br><br>";
+                        }
```

Comment:
Would it be useful to show any message if no tabs were found?

Reply:
Done.